bin/test-browser-example.js

```diff
@@ -3,11 +3,6 @@
 
 import { execa } from 'execa'
 
```

Comment:
try {
  // install deps
  await execa('npx', ['-y', 'playwright', 'install', '--with-deps'], {
    stdio: 'inherit'
  })
} catch (err) {
  if (!err.message.includes('Could not get lock')) {
    throw err
  }
}